datamade/nwss-data-standard
# Finalize release changes

## Overview

This PR updates the README and adds a block in CI to push tags to PyPI.

Closes #14.

## Testing Instructions

* Confirm CI passes.
* Merge, then push a tag. Confirm a release is created in PyPI.
